app/notify/App/Notification/MoveToPylon/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react'
+
+import link from '../../../../../resources/link'
+
+import { Body, Item, Title, PylonConfirm, PylonConfirmButton, PylonConfirmButtonSub } from '../../styled'
+
+const MoveToPylon = () => {
+  return (
+    <>
+      <Title>{'Chain preset updates'}</Title>
+      <Body>
+        <Item>
+          <div>Chains using our built-in Infura & Alchemy presets will</div>
```

Comment:
I think a little more context might be useful here. something like

_"We will soon be retiring open access to Infura and Alchemy in favor of our own JSON-RPC proxy called Pylon._

_Pylon allows us to have more control over the quality and privacy of the default connections we offer in Frame._

_All chains using the Infura or Alchemy presets will be migrated to use Pylon going forward. To continue using Infura or Alchemy directly, visit them to create an account and use the "custom" preset."_